### DISCUSSION TITLE
西安交通大学（兴庆校区）

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#32] [University]
> 回答来自群友

**宿舍是上床下桌吗**
是上床下桌

**教室和宿舍有没有空调**
都有空调，空调够冷

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
四人寝室独立厕所，浴室是全校公共澡堂（温泉澡堂，一股硫磺味）澡堂离宿舍最少5分钟

**有早自习、晚自习吗**
没有自习，大一有英语晚自习，每周两次

**有晨跑吗**
有晨跑，大一大二要早起打卡

**每学期跑步打卡的要求是多少公里，可以骑车吗**
跑步打卡没有要求，到体育场签到就行了

**寒暑假放多久，每年小学期有多长**
寒暑假正常，小学期是寒暑假前一个月，不上直接回家

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
允许外卖，外卖不太远3分钟左右

**学校附近有地铁站吗**
有地铁站20分钟

**宿舍楼有洗衣机吗**
有洗衣机，很脏不建议用，我因为洗衣服经常过敏

**校园网怎么样**
校园网速度不快勉强能用，大部分同学办电信网

**每天断电断网吗，几点开始断**
12点断电，不断网，可以接空调电

**食堂价格贵吗，会吃出异物吗**
食堂不贵，不好吃，吃出异物不至于

**洗澡热水供应时间**
洗澡好像到21点

**校园内可以骑电瓶车吗，电池在哪能充电**
不知道怎么电瓶车

**宿舍限电情况**
宿舍好像限功率，但是具体不清楚

**通宵自习有去处吗**
通宵自习图书馆就行

**大一能带电脑吗**
大一能带电脑

**Comment by callG** [comment]
澡堂里宿舍最少 5 分钟 -> 澡堂离宿舍最少 5 分钟？

**Reply by YanWQ-monad** [reply]
应该是的，搬运时没仔细看 😂

**Reply by 029030** [reply]
这条得改改，现在有一部分的楼是自带浴室了，不过大部分老楼依然是公共浴室，走过去倒也没有5分钟吧，最远的楼是5分钟差不多

**Comment by 029030** [comment]
把新生群问题集合发一下，供大家参考
1. 军训尺寸预定
答：军训服装不用预订，开学后现场领取，现场选尺寸，尽量一次性解决。（8月11日答）8
2. 班级与班级群问题
答：班级可以再http：//hello.xjtu.edu.cn/里登录查询，学号在录取8通知书的条形码上。&
班级群可以看群里在线文档第二个就是。其中没填大概率b就是没建立。其中电信2202好像群号错误（8月12答）
3. 越杰计划、钱班等
答：入学指南19页。（8月11日答）
4. 安全教育课程”
答：下载移动交大app，进入迎新离校，下方第9一6行第个，先做试卷摸底然后看视频，视频要一直看到有完成的提示出来再关闭。花费时大概是50min能k做先做。可能存在没有立即更新的情况（8月11日答）
5. 宿舍问
答：书院是′住1宿的地方，一般是相同专业在一个宿舍，专业对应书院参考入学指南14页。女生似乎是有可能不同专业一起住彩虹楼。8月11日答）
6. 军训
答：20号晚上开始。（8月11日答）
7. 银行卡
答：没有随录取通知书发送，要到西安交通大学报到时领取，是中国银行的卡，还要自8月11日答
8. 陕西一码通
答：是微信小程序。先根据：西西安市碑林区太乙路街道咸宁西路28号填写，小区应该是填西安交通大学兴庆校区（大部分学生，小部分在创新港），楼号就写东/西 xx，楼排就写0，门牌写宿舍号。也可以先乱填后以改。（8月15日答）
9.买教材））
答：一般不建议直接预定教材（因为会额外买很多无用的书）而且这一次只是买大一的书。一般建议直接找直系学长学姐购买，或者学校周围二手书店。@（8月11日答）
答：这是学校政策问题，先等学校或者辅导员通知。目前是迎新志愿者要三天两检。（8月12日答
11.报道时家长能进校吗
答：疫情防控原因不允许。行李问题会有对应的迎新的学长学姐帮你。（8月11日答）
12.社团群问题
答：群里群相册-社团群有，请耐心寻找。（8月@@11日答）
13.直接在学校预定被褥问题
答：在选宿舍的同时会附带是否购买床上用品的选择。预定后会直接送到宿舍（8月11日答）
14.请问教材预订里的专业是填高考招生的大类专_业还是学部或者学院呢
答：按21届来看，大类内的课程安排相同，22届至少不会大改。不过今年既然分好学院了，或许填目前分班的专业类比较合适？（8月11日答）
15.入学考试！
答：考试时间暂定9月4日。全员参加。英语、数学。英语是四级难度，试卷与高考试卷不一样（如听力题目不是印在试卷上而是在录音里，而且要自己带收音机和有线耳机）
入学考试作用就是摸底，譬如英语用来分班（A、B、C班，其中A班有提&前考31四级的机会，但是平时考试课业难度更大，B班是大部分人呆的班，就是正常上课，C班就是基础薄弱班，对应平时考试课业难度更低，但是据说全白卷都进不了C（）班？）。
经过入学考试选拔，一部分同学6获得各种钱班/未来/越杰计划的面试资格，其中钱班、未来还需要考物理。钱院班级选拔复试都有面试（英语能力，个人思维）等），此外化生班加试化学生物。其中钱班部分可以换钱院，然后具体考什么见指南（8月18日答）
16.在一个班九个女生七个陕西的情况下有没有可能四个陕西同一寝室？
答：宿舍是自己选的（8月11日答）
17.不住校
答：可以到外面租房子住，但是要提前和老师沟通报备。（8月11日答）
18。澡堂
答：一般是大澡堂，但是有隔间。（8月11日答）
19.宿舍上铺多高？
答：有人手测95cm，但是不同书院有所差别。据说学校里有锯蚊帐腿的地方。（8月12日答）
20.学习与工作经历填什么？
答：﻿和以前填档案一样，填小学、初中、高中入学、毕业时间那些的（8月12日答）；
2﻿1﻿.请问体检单在哪？
答：体检单在预约体检时会有电子版，要在电脑端迎新离校（
http://hello.xjtu.edu.cn/ ）处进入预约体检完成操作后就可以收到电子版，不需要缴费直接打印即可，后面可能是在财务缴费处统一缴费，可以黑白，在校医院进行。（8月13日答）
22. 请问在哪里体检，学校里的医院吗？
答：入学指南13页（8月12日答）
23:学费啥的是现金还是
答：网上缴费吧。移动交大app，迎新离校，财务缴费。（8月12日答）
24.本科绿色通道
答：入学指南第5页（8月12日答）
25.体检预定
答：要求完成在线安全手续，即弄完在线安全教育的视频。如果做完没有显示完成，就重启app，或给客服打电话。移动交大app，新生迎新离校第二行第一个。（8月12日答）
26.照片
答：红蓝底、一寸两寸多少都备一点，并且保存电子版。其实所有重要的文件都可以存电子版，自己保管好。（8月12日答）
27.电器使用问题、最大功率
答：基本煮饭用的不用考虑肯定用不了。如果有特殊情况请联系辅导员（例如保存眼药）。（8月12日答）
28.入社团时间问题
答：先加入招新群，具体面试时间都会在招新群通知，有的社团大一大二都能参加（部分社团有短信通知）（8月12日答）
29.空调问题
答;都有。但是要自行租借。（8月12日答）
30.户口问题
答：不一定要转。客观上，转了没什么好处、也没坏处。而且毕业还要迁出去现在迁入了户口的话是在学校的集体户口（8月12日答））
31.宿舍床位具体情况v
答：宿舍床位大差不差，好像ad两个8里面有一个有插座（或者四个都有吧。只是只有。一个可以装路由器），但是是随机的（8月12日答）
32.办理助学贷款后是必须办绿色通道么
答：入学指南第5页（8月12日答）？
33.收音机要求
答：能接收FM86.1的，要自备，能接收FM86.1即可，但是最好不是旋钮式、不能显示具体频道，最好是有电子屏的2。耳机可以是戴式（8月13日答）
34.几号开学考
答：全员参加，暂定8月24号下午，各科2.5h好像。（8月15日答）
35.档案
答：自带学生档案者，报道后一周内交给所在书院的辅导员l
邮寄档7案者，暂时先邮到 陕西省西安市、碑林区太乙路街道咸宁西路28号西安交通大学学生处档案室，收件人、电话暂时不知道（8月12日答）
36.转团关系
答：进入https://mp.weixin.qq.com/s?__biz=MzIyMzgyNTk5OQ==&mid=2247567890&idx=1&sn=742d837f51a1237fbe94214bcffd0107&chksm=e81bd332df6c5a24ca7d27374061e2488954a8f3b2c1fdee9a2589d4c371b5f267433a551507&mpshare=1&scene=23&srcid=0811svl6FZVPhaOXL7smqkU5&sharer_sharetime=1660182310782&sharer_shareid=dfc9d2048007676f5d406182919b4b5a#rd
如果登不进去请联系所在高中的团委老师解决。具体填报如下

单位地址：陕西省西安市咸宁西路28号
团关系地址：陕西省-省教育-西安交通大学-XX书院-XX班级团支部
班级团支部需要至少9月才能建立，目前高中没有催的先不要转，高中催的话联系书院团口负责老师
（8月12日答）
37.体检前要求
答：体检前8小时要空腹，不能吃饭喝水（8月12日答）
38.军训和入学考安排
答：入学考试时间暂定9月4日。军训晚4上要训练6
时间安排：早上：6：40－7：20（食堂7点开始营业）3
8：00 - 11：30
下午：3：00－5：30
晚上：7：30－9：30（8月15日答)
39.住宿尺寸问题
答：床的尺寸是1.9*0.9 。其他物件具体见入学指南15页（8月12日答）
40.宿舍床垫需要自己购买吗
答;可以自己买也可以买学校的（8月312日答
41..学校快递地址
答：陕西省西安市碑林区太乙路街道咸宁西路28号西安交通大学兴庆校区
西安市长安区高桥街道中国西部科技创新港
可以不用写具体楼号。因为学校只有一个快递中心。据说网购填地址会自动跳出来（8月15日答）
42.有宿舍布局吗（两人血书）
四个床位，靠门AD，靠窗CB，具体左右以及空调位置分布各楼不同（8月12日答）
43.收音机和耳机需要自备吗？如果要，该买什么规格的？
答：要自备（用于英语考试），能接收FM86.1即可，但是最好不是旋钮式或者不能显示具体频道的款式，最好是有电子屏的。耳机可以是头戴式（封闭式），但是不能是无线式，会被判定作弊。若是交过1100的教材费会配送收音机。若是要质量好建议网购，学校周围也可以买的到。（8月14日答）8
44.彩虹楼可以买1.1m高的床
答：可以（8月12日答）
4﻿5﻿.﻿现在可以出入校园吗？
答:新生可以给保安出示电子校园卡进入校，建议东南门。（8月12日答）
46.本科健康日报填不满十四天j能报道么?（算上补填的三天也不够十四天）
答：迎新系统答疑群老师说到时候可以联系辅导员补签（8月12日答）
47.买床帘有必要吗？
答：看自己需求（8月12日答）
48.本地迎新志愿者是否需要三天两检
答：需要（8月12日答）
49.志愿者小哥哥帅吗？
答：都挺帅 心地善良积极活泼（8月12日答）
50.选宿舍的网址
详情见69条（8月13日答）
51.学校有卖收音机的嘛？
答：有卖的（8月12日答）
52.在学校买收音机方便吗？人多不多？性价比高吗
答：有卖的，但考前一般会售空（8月12日答）
53.开学电脑必带吗？平时主要哪些情况需要用电脑？
建议携带，选课、网课、资料查阅、计算机课程需要用电脑（8月12日答）
54.校园网费用本科四年期间只交一次吗？
入网费100就交一次，实际的套餐是自己选的，有按月的也有按年的（8月12日答
55.完善个人档案后还可以再次修改吗？
答：可以。流程是一致的，再次点击进入就行。（8月12日答）
56.加长床具体是怎么操作的
到校后联系宿舍楼一楼宿管办公室（8月12日答）
57.新生是不是只能住一年彩虹楼
分性别，男生与彩虹楼无缘（8月12日答）
58.到校领取银行卡后如何、在哪里激活？
答：去对应银行激活（8月12日答
59.群内提到宿舍是东七是宿舍号单数靠南，请问其他的楼的宿舍号都是怎么排布的呢？
具体看选宿舍网站提供的平面图吧
60.报名当天领卡，交协议书，办理住宿，学院注册手续这些用时大概多久？
答：大概一两个小时（8月12日答）
6﻿1﻿.﻿如果1﻿9﻿号当天报完名，﻿1﻿9﻿号当天还可以出去住嘛？﻿
答：当天应该辅导员在，找到辅导员问。（8月12日答）
62.寝室是不是一定要同班同学
答：不一定，但大概率是，最多是隔壁班的。（8月12日答）
63.校园网申请了可以撤销嘛，﻿想看看其他套餐
答：不可以，入网费100交一次就可以，剩下的学期内（或月内）套餐应该是不可以的，想换可以用完在换其他的
64.校园卡能不能不用高考照片
答：大概率不能（8月12日答）
65 军训怎么分旅、营这些
答：全校编为军训旅；按照班级将某几个班级编为一个营（8月12日答）
6﻿6﻿ ﻿新生到达西安进行三天两检的三天期间在未完成两检时可以进入学校体检并办理手续吗
体检校医院那应该不会查，但门口保安可能会查，进不去的话在新生群问问别人想想办法找个查得松点的门混进去或者在哪翻墙比较轻松，我只是略有耳闻但具体不知道
67.钱班是只有这一次有机会进，﻿还是每个学期都可以
答：钱班的各种事务在这个网站查（钱学森书院官方网站）http://bjb.xjtu.edu.cn/
这边好像看到计算机H可以大一再转入，但是要补修大一未学的内容。（8月12日答）
68.选宿舍建议
答：具体看需求。思考方向就是有网线接口能有线上网，身体不好选远离空调/靠门（离空调近的一般是BC，对应的远离空调就是AD）（8月13日答）
答：目前似乎有未完成安全教育的视频不能选宿舍的规定（？）。移动交大6app选择宿舍会显示直接在自选宿舍系统选择，就在这个网址进去里选自选宿舍http://housing.xjtu.edu.cn/dmWeb
69.自选宿舍
13号中午12点后进行的是收藏宿舍的操作，相当于加入购物车，14号9点后按照对应时间批次(存在骗老实人的嫌疑）进行抢购具体时间参见下图。（8月13日答）

69.本人名字不太好听，上了大学还能改名吗？
答：参见入学指南第12页（8月13日答）
70.需要买那些教材？
答：私聊自己的直系学长学姐问。（8月13日答）
71.宿舍的﻿空调﻿在﻿哪个位置﻿（﻿吹空调﻿难受꒦ິ^꒦ິ﻿
答：见问题68（8月13日答）
72.兵役登记
答：只有男性公民需要办理。找对应社区工作人员办理。开学最好带来兵役登记证，虽然有学长说去年没查。（8月16日答）
73.刚刚做完安全教育后其他东西还是不能做体检预约等
答：可能是系统还没更新，要等半小时反应。实在不行再问客服。（8月13日答）
74.彩虹楼离快递站点近吗？19号快递站人多吗？
答：和东区比还是近的（8月14日答）
75.选宿舍时的床具里包不包含床垫
答：有床褥（8月14日答
76.宿舍的3号床与2号床有啥区别？
答：具体得分楼看（8月14日答）
77.西21是彩虹楼吗？有电梯吗？
答：是彩虹楼，有（8月14日答）
78.网费现在可以缴吗？该怎么缴？
答：老师说最好开学后交，想现在交的话百度西安交通大学网络信息中心，打开官网后点击网络服务中的缴费平台，点击常见问题并打开提供的网址，登陆后看右下角的学生宿舍入网流程（8月14日答）
79.每日填写健康报告是在哪里填的啊？
答：移动交通大学APP，校园生活里有健康日报（8月14日答）
80.不会出现男女同一楼层混住的情况吧
答：不会（8月14日答） 仲英7﻿层会
81.哪个床位会靠厕所？﻿（﻿1﻿、﻿2﻿、﻿3﻿、﻿4﻿几个床位来看）﻿
答：看平面图，各楼不同（8月14日答）
82.西交APP在线安全教育怎么算完成？﻿
答：完成所有任务后等半小时（8月14日答）
83.已经进行了生源地信用助学贷款是否还需要办理绿色通道
答：也可以吧，一个是给你钱的，一个是延迟收你钱，不矛盾吧（8月14日答）
84.开学报到那两天取快递方便吗？往年开学寄快递的人多不多？
答：多（8月14日答）
85.学校超市卖盆和暖水瓶之类的吗？
答：卖的，就算买不到，学校身处市区买东西的地方还是很多的（8月14日答）
86.宿舍楼层平面图在哪里查看
答：选宿舍网站点击楼号和房号之间那个表示楼层的数字（8月14日答）
87.国庆放假吗？可以回家吗？
答：正常放（8月14日答）
88.西安最近快递怎么样，受疫情影响么
答：目前是正常的（8月14日答）
89.﻿1﻿1﻿0﻿0﻿元“﻿教材付款”﻿填的专业是填一开始大类的录取专业名称还是分完学院后的学院名称
答：我记得上面写的专业吧（8月14日答）
90.东七是奇数房间朝阳还是偶数朝阳
答：看选宿舍网站平面图（见问题86），有方向标的（8月14日答）
91.寝室要是有不住校的是会变成三人寝还是另外安排？
答：一个班分的宿舍是固定的，学校层面应该不会安排，个人调整宿舍倒是有可能（8月14日答）
92.西21-23舍都是彩虹楼吗？只能住一年吗
答：入住时长这个每年学校决策不一定，建议开学问老师，只能说排除性别原因宿舍变动不大（8月14日答）
93.一个班有多少人？
答：二三十（8月14日答）
94.宿舍安床帘的话应该是多大尺寸的（或者选择）
答：目前普遍是说1.1m高，但是有的楼层会多一条梁影响高度到0.9m。用挂的窗帘需要在天花板加钩子。最好买有伸缩架的，但是也要注意伸缩范围。所以一般建议到了实地测量后，再买不迟。（8月14日答）
95.学生选定宿舍后可以和其他人换宿吗
答：可以和辅导员报备商量。据说有无法忍受舍友换宿舍的先例。（8月14日答）
96.未来技术学院选拔竞争压力如何？
答：报考的勇气也是考核的一部分。
97.鞋架
答：没有鞋架。不如放地上？（8月14日答）
98.二手书购买渠道（
答：二手书交流群（qq群：629331432）、学校周边二手书店、直接联系直系学长学姐购买、教材大礼包。（8月14日答）
99.排插（插线板））
答：建议挂在墙上。长度尽量买长一点（例如3m）（8月14日答）
100.可以不加入任何社团吗？
答：可以，完全遵循自愿原则。（8月14日答）
101.防晒喷雾可以带上高铁吗
答：150毫升以下可以。（8月14日答）
1﻿0﻿2﻿.﻿1﻿8﻿号可以进宿舍放东西吗
答：可以（8月18日答）
103.在几个版本的校园地图找了一圈东和西1-23楼，唯独没看见东4，请问东4楼在哪
答:东四楼就是东21.想了解原因请自行百度。（8月18日答）
104.请问19号报道的话是有西安48小时内核酸就可以进学校还是要西安三天两检结果出来才可以（如果三天两检才可以的话最迟17号就得去，对吧？）

105.教材预定支付不了
答：西安银行无法支付。可以试一试保存二维码去微信扫（但是据说会说账号有风险不能交易）（8月14日答）
106.宿舍允许自行改造吗？比如贴墙纸、安装到顶滑轨床帘什么的
答：可以的（8月18日答）
107.大学还需要用到钉钉吗？想卸了
答：？随便卸
（补充：如果高数是贾慧莲老师，大概率会用到钉钉）（8月18日答）
108.钱院之类的班级选拔
答：请关注钱学森学院官网和微信公众号。
钱院新生选拔指南https://mp.weixin.qq.com/s?__biz=MjM5NzI4NDY0Mg==&mid=2651307735&idx=1&sn=1a233f60af789a1b0aa64ff38464f375&chksm=bd2f409a8a58c98ca665fd3d391fdac2b0bab40060a1ab151eb7ee106b5f6f23fca8eb4ad2bb&mpshare=1&scene=23&srcid=081547JvSNFPUhtEzOr2Dwxn&sharer_sharetime=1660530627377&sharer_shareid=09ae6e1b69752b01c62b4a098015ea4e#rd
钱学森书院官方网站[http://bjb.xjtu.edu.cn/（8月18日答）](http://bjb.xjtu.edu.cn/%EF%BC%888%E6%9C%8818%E6%97%A5%E7%AD%94%EF%BC%89)
109.哪个专业xjj好看
答：都好看，xgg也好看哦（8月18日答）
1﻿1﻿0﻿.﻿社团可以加入几个？﻿
答：量力而行。（8月18日答）
111.彩虹楼（西23）晚上有门禁吗，断网断电吗

112.团组织关系转接的介绍信的转入怎么填?
答：别着急嘛，可以等团支书通知（8月18日答）
1﻿1﻿3﻿.有推荐的入学考试收音机型号吗？学长学姐用后感觉不错的都推一下
答：见48点。（8月18日答）
114.必须按入学指南上先报到后体检吗？那18号预约了体检能不能报到？
19号及以后报道，不一定分先后（8月18日答）
115.体检单需要提前自行打印吗？在哪里打印？
答:前面问题已经说过了，罚你再看一遍。（8月18日答）
116.三方协议日期写哪天？
答：应该就写填的那一天吧（8月18日答）
117.军训服装
答：学校会发新的衣服和靴子。衣服包括短袖、外套和长裤。靴子建议自备软的鞋垫。（8月15日答）
118.报道顺序
答：据说可以先体检再报道（8月18日答）
119.报道过程需用到身份证吗
答：你上大学都用得到身份证的。（8月18日答）
120.预定的报道前一天18号的体检，请问到时候是只带体检单和身份证直接去校医院体检吗？
通知要求带体检单和校园卡，但18号是报道领卡前一天，这个问题现在在等学校回复，新生群里有法是只带体检单就行，当然进门要查两码一酸(详见西安交通大学招生办微信公众号8月16日发布的《西安交通大学2022级本科新生入校安排的紧急通知》）（8月18日答）
121.18号能进宿舍放行李吗？能从西南门进入吗？如果能进核酸要求有啥要求吗？
新生群里说可以，后两个问看西安交通大学招生办微信公众号8月16日发布的《西安交通大学2022级本科新生入校安排的紧急通知》（8月18日答）
122.19号行李可以出来拿着两次吗
如果你问的是能不能反复进出校门，是可以的（8月18日答）
123.新生群里有宿管说18号持有学校的入住通知单就可以办理入住，请问入住通知单是什么
持校园卡办理，对于提前入校等未领到校园卡的情况，可以持入学通知书和身份证办理入住（8月18日答）
124.请问有没有什么可以加的QQ，比如各种墙以及貌似有个什么生活服务站？
交大表白墙1264900445
仙交表白墙1956044111（8月18日答）
125.要求带的户口本复印件是所有学生必须带还是只有迁户口的学生要带
不迁户口不用带户口本、户口本复印件（8月18日答）
126.是不是就可以18号去体检去取快递了，请问取快递的流程是什么
答：新生群里说能进校就能取快递；所有快递点均在西15快递街，可以先下载“菜鸟”APP，部分快递点扫描身份码取件、其余凭取件码取件（8月18日答）
1﻿2﻿7﻿.﻿校园里能穿拖鞋吗？
答：啊可以可以（8月18日答）
128.健康承诺书
答：健康承诺书是在新生报道信息采集系统里填写完后会跳出一个电子签名，就是那个。
[https://jinshuju.net/f/Ad9a7w如果看到有word版的承诺书如果没有西安交通大学抬头应该就不是。（8月18日答）](https://jinshuju.net/f/Ad9a7w%E5%A6%82%E6%9E%9C%E7%9C%8B%E5%88%B0%E6%9C%89word%E7%89%88%E7%9A%84%E6%89%BF%E8%AF%BA%E4%B9%A6%E5%A6%82%E6%9E%9C%E6%B2%A1%E6%9C%89%E8%A5%BF%E5%AE%89%E4%BA%A4%E9%80%9A%E5%A4%A7%E5%AD%A6%E6%8A%AC%E5%A4%B4%E5%BA%94%E8%AF%A5%E5%B0%B1%E4%B8%8D%E6%98%AF%E3%80%82%EF%BC%888%E6%9C%8818%E6%97%A5%E7%AD%94%EF%BC%89)
129.预约完成后为什么收不到体检单？
答：登录网页端迎新系统下载并打印（8月18日答）
130.教材预订的1100元是可以多退少补的是么？也就是说我不领的部分可以退钱给我？
答：会把所有书打包好（包括没用的书），直接带回去,这里多退少补是因为各专业用书不同（8月18日答）
131.校园里哪个运营商信号最好？
电信>联通>移动（8月18日答）
132.学校超市付款可以用校园卡吗？
答：可以（8月18日答）
133.宿舍信息查询
答：宿舍信息查询网站 http://room.xjtu.edu.cn/#/selectbedshow （8月18日答）
134.18号到后在哪儿办理入住通知单？
见123（8月18日答）
135.报道可以分两天安排嘛？
可以（8月18日答）
136. 没法按时报到参加开学考会有什么影响？
开学考延期至9月4号。但是钱院什么的宣讲还是这几天，具体见钱学森学院官网。（8月18日答）
137.开学健康信息监测及承诺书需要打印并在报到时用到吗？
见128点（8月18日答）
138.现在疫情形势如何
答：挺严峻的
1﻿3﻿9﻿.﻿校园卡的人脸图片可以更换吗？
答：
140.预约好的体检怎么办？（瘫
答：体检推迟喽（8月18日答）
141.明天不去学校，但快递明天就到学校了怎么办？
答：快递一般保存3天，快递存放时间过长会有人打电话给你你说明情况即可。或者你和对应书院的迎新学长学姐/学生会说明情况代领。或者找信得过的学长学姐代领。
最新消息：在保卫处登记好名字书院宿舍，快递和行李贴好标签志愿者送到书院，然后书院志愿者送到宿舍（8月18日答）
142.西安市新生入校
答：18号西安市本地学生不能入校。（？）但是已经有人入校了（8月18日答）
143.体育一、体育三
答：一个大一上课，一个大二上课。难度据说没区别。（8月18日答）
144.学校宽带
答：一个宿舍似乎只有一个网线接口（可以接交换机多接几个）宿舍的解决方案最好的应该一个人买然后接路由器。微信公众号XJTUANA里有使用学校网络的具体流程。套餐订购网址http://nethelp.xjtu.edu.cn/selfservice/main.php （8月18日答）
145.用水问题
答：请自带暖壶/水壶，水房（打水的地方）每一层楼都有（8月18日答）
146.学生卡
答：校园卡在书院的迎新点领取。初始密码6个8充值：支付宝搜索“校园一卡通”，点击第一个程序，输入自己的学校（记得改所在地成西安）），姓名，学号就能充值了，移动交大app比较慢可以多刷新几次就看到自己的余额变化了，绑定银行卡要到教务处办理。记住先去超市用学生卡买东西才能用卡洗澡！！！（8月18日答）
147.新生入住指南
答：包括入住手续、卧具、饮水机、浴室洗浴、洗涤、空调报修、钥匙、卫生、用电（8月18日答）6
空调费用是一个宿舍一起担。
148.学校床上用具
答：预定后，用具直接送到宿舍。缴费在思源中心西门现场缴费。应该包括两套。一套用于洗。没有预定也可以在思源中心西门现场补买。买到后记得晒一晒。（8月18日答）
149.请问哪些社团可以加学分？
没有加学分这种说法。要加德育分，就多做做志愿者
150.缴纳学费
答：流程见下6https://mp.weixin.qq.com/s?__biz=MzI5NzE3MTQ5NA==&mid=2650282302&idx=1&sn=3841665d445593035a1e7cff7f48fa76&chksm=f4b59677c3c21f610cb75936b3b43a9b66051ffad97be30e65926b83facba5eee860f96f4d74&mpshare=1&scene=23&srcid=0817b6vUrzI6cZqn3X8FQsiE&sharer_sharetime=1660733145908&sharer_shareid=24e94e1381eb020d7254d8ad6567bb35#rd
151.计算机
答：计算机1是c语言，计算机3是python，医学学习python。
152.选课问题（时间、方式）
8月31日第一轮选课。具体见教务处网站。
教务处网站http://dean.xjtu.edu.cn/
选课通知http://dean.xjtu.edu.cn/info/1015/11427.htm
153.现在网购的话快递还能正常进校吗（8.20）
有部分快递会被卡（从高风险寄出），其他的基本没问题。
154.快递站都有哪些快递
基本都有
154.京东同城速配是送到学校快递街吗？要在地址上标注学校地址以外的东西吗（如宿舍楼）？
答：你是说外卖吗？外卖地址外卖地址陕西省西安市碑林区咸宁西路28号西安交通大学西15舍下不管写不写最后都会集中在西十五
155.暖壶/水壶学校有卖吗
答：康三超市有
156.如何提前准备选课

157求群文件里英语水平考试样题的对应听力音频
答：那份样卷据说与正式试卷安排有差异。建议直接找四级真题或者模拟题进行练习
158.快递打电话说到了等开学取但是没有取件码怎么办?

159.迎新离校，财政缴费可是怎么缴费呀，没有二维码什么之类的，点不动
2022年新生缴费指南https://mp.weixin.qq.com/s?__biz=MzI5NzE3MTQ5NA==&mid=2650282302&idx=1&sn=3841665d445593035a1e7cff7f48fa76&chksm=f4b59677c3c21f610cb75936b3b43a9b66051ffad97be30e65926b83facba5eee860f96f4d74&mpshare=1&scene=23&srcid=0817b6vUrzI6cZqn3X8FQsiE&sharer_sharetime=1660733145908&sharer_shareid=24e94e1381eb020d7254d8ad6567bb35#rd
160.体检是在校医院吗（校内吗），是先只拿体检单什么的去体检然后再拿行李报到吗
答：是在校医院。体检报道不冲突
161.学校有单独购买新书的地方吗
答：如果是教材的话，教材中心去问问。如果是课外书网购不好吗
162.选课问题（如何查看主修要选啥）
选课问题
答：首先认真阅读教务处文件http://dean.xjtu.edu.cn/info/1015/11427.htm （很重要！）（包括学分要求，选课各个批次的目的、方式、结果等等内容）
然后看群文件附件1.（找到自己专业有指导方案）
附件2.（是讲英语的选课，具体见文件和教务处文件）
附件3.（体育选课，据说体育1和体育3只是上课一个在大一一个在大二）
附件4.
4.1课程条目分类：包括是核心课还是选修课，选修课和核心课的要求看教务出那个文件，以及是什么模块的（模块要求各学院不同，具体参见附件4.3，包括总体要求和学院要求）
4.2选修课课程具体信息：比较要注意的是该课程的学分（上课的数量就是由学分决定的，大学四年（尽量别拖太迟合理安排/不然无法毕业）修满学分即可，这个学分要求问题看教务处的文件）西安交通大学通识教育选课指南（核心课介绍） https://flbook.mwkj.net/c/bG2pJlN5Ls
4.3课程选择要求：一定要仔细看文件前面的内容的文字和后面表格！！！
PS1.本科教务中我的方案中的学分分配仅作参考，具体以教务处通知为准
PS2.课程时间冲突会有提示选不上的
